### PR TITLE
fixed Style Guide issues, streamer mode detection errors, and removed unnecessary code.

### PR DESCRIPTION
**Pull request by raphael069**

### CODE REVIEW

**Review by Mark7888 — CHANGES_REQUESTED** [review]
Getting better, but the function `streamer_mode_status` still has two minor problems.
- Place it inside the `Sonar` class, in my opinion, it belongs there.
- It returns a boolean (`True` or `False`). Rename it to represent an action of getting the state of streamer mode, like `is_streamer_mode_enabled` or just `is_streamer_mode`.

**Comment by raphael069** [comment]
> * is_streamer_mode

function is now called "is_streamer_mode" and in the Sonar class.

File: src/steelseries_sonar_py/sonar.py [hunk 1]
```diff
@@ -16,19 +17,47 @@ class Sonar:
...
-
         if app_data_path is not None:
             self.app_data_path = app_data_path
 
         self.load_base_url()
         self.load_server_address()
 
+        if streamer_mode is None:
+            # get the streamer mode using a request to the server
+            self.streamer_mode = Sonar.is_streamer_mode(self)
```
Comment (Mark7888) [inline src/steelseries_sonar_py/sonar.py]: The function is now inside the class, so you can call it like this: `self.streamer_mode = self.is_streamer_mode()`

**Review by Mark7888 — CHANGES_REQUESTED** [review]

File: src/steelseries_sonar_py/sonar.py (hunk 1 above)
Comment (raphael069) [reply src/steelseries_sonar_py/sonar.py]: Yes, self. is simpler.

**Comment by Mark7888** [comment]
Yes, this gonna be good, but remove the self inside the function call. You only need the self variable when defining it `def is_streamer_mode(self):`, when calling it with `self.is_streamer_mode()`, the object itself provides the value of `self`.

Here the answers explain why:
https://stackoverflow.com/questions/18679803/python-calling-method-without-self

**Review by Mark7888 — CHANGES_REQUESTED** [review]
.

**Comment by raphael069** [comment]
> Yes, this gonna be good, but remove the self inside the function call. You only need the self variable when defining it `def is_streamer_mode(self):`, when calling it with `self.is_streamer_mode()`, the object itself provides the value of `self`.
> 
> Here the answers explain why: https://stackoverflow.com/questions/18679803/python-calling-method-without-self

I´m actually aware of this behavior looks like i missed it while fixing the self. / Sonar. call

**Comment by Mark7888** [comment]
Nice. I'm merging this now, and will take one more look at it Saturday, (when I can test it on my Windows machine), and then will publish a release.

**Pull request merged**